src/pages/components/PageDetailsPage/PageDetailsPage.tsx

```diff
@@ -142,148 +142,156 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
       {({
         change,
         data,
+        validationErrors,
         handlers,
         submit,
-        isSaveDisabled,
         attributeRichTextGetters,
-      }) => (
-        <Container>
-          <Backlink href={pageListUrl()}>
-            {intl.formatMessage(sectionNames.pages)}
-          </Backlink>
-          <PageHeader
-            title={
-              !pageExists
-                ? intl.formatMessage({
-                    id: "gr53VQ",
-                    defaultMessage: "Create Page",
-                    description: "page header",
-                  })
-                : page?.title
-            }
-          />
-          <Grid>
-            <div>
-              <PageInfo
-                data={data}
-                disabled={loading}
-                errors={errors}
-                onChange={change}
-              />
-              <CardSpacer />
-              <SeoForm
-                errors={errors}
-                allowEmptySlug={!pageExists}
-                description={data.seoDescription}
-                disabled={loading}
-                descriptionPlaceholder={""} // TODO: Cast description to string and trim it
-                onChange={change}
-                slug={data.slug}
-                slugPlaceholder={data.title}
-                title={data.seoTitle}
-                titlePlaceholder={data.title}
-                helperText={intl.formatMessage({
-                  id: "jZbT0O",
-                  defaultMessage:
-                    "Add search engine title and description to make this page easier to find",
-                })}
-              />
-              <CardSpacer />
-              {data.attributes.length > 0 && (
-                <Attributes
-                  attributes={data.attributes}
-                  attributeValues={attributeValues}
+      }) => {
+        const errors = [...apiErrors, ...validationErrors];
+
+        return (
+          <Container>
+            <Backlink href={pageListUrl()}>
+              {intl.formatMessage(sectionNames.pages)}
+            </Backlink>
+            <PageHeader
+              title={
+                !pageExists
+                  ? intl.formatMessage({
```

Comment:
let's keep translations in the messages file as in the other places, worth following that rule, consistency.